src/Classes/ServiceAPI/MyRadio_Timeslot.php

```diff
@@ -373,8 +373,15 @@ public function toDataSource($mixins = [])
                 'start_time' => CoreUtils::happyTime($this->getStartTime()),
                 'duration' => $this->getDuration(),
                 'mixcloud_status' => $this->getMeta('upload_state'),
+                //PLEASE REMOVE THIS COMMENTED WHEN USED IN PRODUCTION
+                
                 'mixcloud_starttime' => $this->getMeta('upload_starttime'),
                 'mixcloud_endtime' => $this->getMeta('upload_endtime'),
+                
+                //PLEASE REMOVE THIS COMMENTED WHEN USED IN DEVELOPMENT
+
+                //'mixcloud_starttime' => $this->getMeta('description'),
+                //'mixcloud_endtime' => $this->getMeta('description'),
```

Comment:
The instructions here aren't very clear. And it might be worth mentioning this in the readme so it's easier to find in future.

Reply:
I was thinking about writing up the guide on how to setup your own mini ury. I'll add it to this pr

Reply:
would do think of my new install.md fixes

Reply:
ah after poking around with your hint of metadata table I managed work out values required for the patch to work. After testing with my 2016-site. The fix appears to work, thank you very much for help I couldn't find it without your help.
I've taken values we use on ury's myradio as they make the most sense for the INSERT INTO :3

Added the following to patch 19.sql which is also used for nname member update
`
INSERT INTO metadata.metadata_key VALUES (20,'upload_starttime',false,'In the case where a manual upload is required (because an event started late) will need to start late. This is a UTC time.',300,false);    
INSERT INTO metadata.metadata_key VALUES (21,'upload_endtime',false,'In the case where a manual upload is required (because an event started late) will need to finish early/late.',300,false);
`